gdnative-core/src/globalscope.rs

```diff
@@ -298,22 +313,22 @@ pub fn stepify(mut value: f32, step: f32) -> f32 {
 /// //Infinite loop between 5.0 and 9.9
 /// let value = 1.5;
 /// let angle = 0.70707;
-/// let value = wrapf(value + 0.1, 5.0, 10.0);
+/// let value = wrapf(value + 0.1, 5.0..10.0);
 /// //Infinite rotation (in radians)
-/// let angle = wrapf(angle + 0.1, 0.0, TAU);
+/// let angle = wrapf(angle + 0.1, 0.0..TAU);
 /// //Infinite rotation (in radians)
-/// let angle = wrapf(angle + 0.1, -PI, PI);
+/// let angle = wrapf(angle + 0.1, -PI..PI);
 /// ```
```

Comment:
Maybe add some `assert_eq!` statements, so this becomes a doctest.

Reply:
uh i thought we don't need them since i added doctests, 
i will add doctest for wrapf and wrapi

Reply:
This `/// ` inline code _is_ a doctest, but they still need `assert_eq!` if you want to have something validated, see [here](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html) 😉